docs/source/worker.rst

```diff
@@ -143,12 +146,13 @@ Memory Management
 
 Workers are given a target memory limit to stay under with the
 command line ``--memory-limit`` keyword or the ``memory_limit=`` Python
-keyword argument.::
+keyword argument, which sets the memory limit per worker processes launched
+by dask-workder ::
 
-    $ dask-worker tcp://scheduler:port --memory-limit=auto  # total available RAM
-    $ dask-worker tcp://scheduler:port --memory-limit=4e9  # four gigabytes
+    $ dask-worker tcp://scheduler:port --memory-limit=auto  # total available RAM on the node
```

Comment:
I recommend using the term "machine" rather than "node" which is often used to mean other things as well.

Reply:
Yes. We shall make sure the set of terminology is defined, if not yet.